examples/python/nv12/README.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Template
+tags: [kebab-case, comma, separated]
+description: "Short ~100-sign description of the example. No longer than 130 signs!"
+python: https://github.com/rerun-io/rerun/tree/latest/examples/python/template/main.py
+rust: https://github.com/rerun-io/rerun/tree/latest/examples/rust/template/src/main.rs
+---
```

Comment:
thanks!

I'm checking with the team if the missing screenshots will cause issues with the website or something. Don't know actually.
But the rest of these fields have to be filled out now at least ;)

Reply:
Oh, missed them completely, thanks!

Reply:
I'm being told it doesn't automatically show up on the rerun.io/examples. So then I'm good with images missing etc. :)